management_system/tests/test_views/test_cook_view.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from management_system.models import Cook
+
+
+class PrivateCookViewTestCase(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="test-user",
+            password="test1234",
+        )
+        self.client.force_login(self.user)
+
+    def test_retrieve_cook_list_view(self):
+        """
+        Test that checks if the cook list view is returned for the logged-in user
+        :return:
+        """
+        cook_1 = Cook.objects.create(
```

Comment:
add more cooks and use `bulk_create` method

docs https://docs.djangoproject.com/en/5.1/ref/models/querysets/#bulk-create